server/lib/nl/fulfillment/base.py

```diff
@@ -55,7 +55,7 @@ def populate_charts(state: PopulateState) -> bool:
   places = state.uttr.places
   handle_contained_in_type(state, places)
 
-  if not state.uttr.svs:
+  if not state.uttr.svs and not state.uttr.multi_svs:
```

Comment:
Should we check `state.chart_vars_map` instead?

Also, do the same in line 68?

Reply:
ah yes, that makes sense, updated